assets/js/modules/analytics/components/dashboard/DashboardUniqueVisitorsWidget.stories.js

```diff
@@ -0,0 +1,215 @@
+		dispatch( STORE_NAME ).receiveGetReport( getAnalyticsMockResponse( optionsEntityURL ), { options: optionsEntityURL } );
+	},
+};
+
+export const LoadingEntityURL = WidgetAreaTemplate.bind();
+LoadingEntityURL.storyName = 'Loading with entity URL set';
+LoadingEntityURL.args = {
+	setupRegistry: ( registry ) => {
+		const { dispatch } = registry;
+
+		provideSiteInfo( registry, { currentEntityURL } );
+		dispatch( STORE_NAME ).receiveGetReport( getAnalyticsMockResponse( optionsCompareEntityURL ), { options: optionsCompareEntityURL } );
+		dispatch( STORE_NAME ).receiveGetReport( getAnalyticsMockResponse( optionsEntityURL ), { options: optionsEntityURL } );
+		dispatch( STORE_NAME ).startResolution( 'getReport', [ optionsEntityURL ] );
+		dispatch( STORE_NAME ).startResolution( 'getReport', [ optionsCompareEntityURL ] );
+	},
+};
+
+export const DataUnavailableEntityURL = WidgetAreaTemplate.bind();
+DataUnavailableEntityURL.storyName = 'Data Unavailable with entity URL set';
+DataUnavailableEntityURL.args = {
+	setupRegistry: ( registry ) => {
+		const { dispatch } = registry;
+
+		provideSiteInfo( registry, { currentEntityURL } );
+		dispatch( STORE_NAME ).receiveGetReport( [], { options: optionsEntityURL } );
+	},
+};
+
+export const ErrorEntityURL = WidgetAreaTemplate.bind();
+ErrorEntityURL.storyName = 'Error with entity URL set';
+ErrorEntityURL.args = {
+	setupRegistry: ( registry ) => {
+		const { dispatch } = registry;
+		const error = {
+			code: 'missing_required_param',
+			message: 'Request parameter is empty: metrics.',
+			data: {},
+		};
+
+		provideSiteInfo( registry, { currentEntityURL } );
+		dispatch( STORE_NAME ).receiveError( error, 'getReport', [ optionsEntityURL ] );
+		dispatch( STORE_NAME ).finishResolution( 'getReport', [ optionsEntityURL ] );
+	},
+};
+
+export default {
+	title: 'Modules/Analytics/Widgets/DashboardUniqueVisitorsWidget',
+	decorators: [
+		( Story ) => {
+			const setupRegistry = ( registry ) => {
+				const { dispatch } = registry;
+
+				const [ property ] = accountsPropertiesProfiles.properties;
+				registry.dispatch( STORE_NAME ).receiveGetSettings( {
+					// eslint-disable-next-line sitekit/acronym-case
+					accountID: property.accountId,
+					// eslint-disable-next-line sitekit/acronym-case
+					internalWebPropertyID: property.internalWebPropertyId,
+					// eslint-disable-next-line sitekit/acronym-case
+					profileID: property.defaultProfileId,
+				} );
+
+				dispatch( CORE_WIDGETS ).registerWidgetArea( areaName, {
+					title: 'Overview',
+					style: WIDGET_AREA_STYLES.BOXES,
+				} );
+				dispatch( CORE_WIDGETS ).registerWidget( widgetSlug, {
+					Component: DashboardUniqueVisitorsWidget,
+					width: WIDGET_WIDTHS.FULL,
+				} );
+				dispatch( CORE_WIDGETS ).assignWidget( widgetSlug, areaName );
+				dispatch( CORE_USER ).setReferenceDate( '2020-09-08' );
+
+				provideModules( registry, [ {
+					active: true,
+					connected: true,
+					slug: 'analytics',
+				} ] );
```

Comment:
If we weren't rendering a widget area (+widget renderer internally), this part probably wouldn't be needed either.

Reply:
@aaemnnosttv I do see we are using the `withActive` HOC in the `DashboardUniqueVisitorsWidget` component. Thus if the module is not loaded, nothing will be displayed: https://github.com/google/site-kit-wp/blob/d6ea05554e20353df9e770abd193c98b006cdf79/assets/js/util/when-active.js#L57-L60

Reply:
Ah yes, I forgot about that HOC we use with most widgets 👍